oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/VersionGarbageCollector.java

```diff
@@ -1092,61 +1124,63 @@ public void collectGarbage(final NodeDocument doc, final GCPhases phases) {
             }
 
             if (!isDeletedOrOrphanedNode(traversedState, greatestExistingAncestorOrSelf, phases, doc)) {
+
                 // here the node is not orphaned which means that we can reach the node from root
-                switch(fullGcMode) {
-                    case NONE : {
+                switch (fullGcMode) {
+                    case NONE: {
```

Comment:
please avoid unrelated reformatting

Reply:
(same applies to other case statements below)

Reply:
Actually, this is the suggested way to format switch statements, in fact it was automatically suggested by the IntelliJ Java text editor.
Also, other switch statements in the Oak project use this formatting, with no space between the value and ":".

Reply:
still, this in unrelated and we should not increase PR change surface when that is not necessary or part of the PR